recipes/makeheaders/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set version = "2018.1.28" %}
+{% set sha256 = "5eeb2b83357a4affefcf48a9d5624c1373a0cfd40d026306d1c2933888a145b0" %}
+{% set commit = "2d5bd5f81a6484c9d6af2542f83bf1f10c7c4f62" %}
+
+package:
+  name: makeheaders-rowanthorpe
+  version: {{ version }}
+
+source:
+  fn: makeheaders-{{ version }}.src.tar.xz
+  url: https://github.com/rowanthorpe/makeheaders/archive/{{ commit }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - gh_1.patch
+
+build:
+  number: 0
+  skip: true  # [win and py<35]
```

Comment:
Should be `py<=35`? No need to build on both `py35` and `py36`. Also, does it not work on VC9 or do you just not want to bother?